### PR TITLE
Add more logging to CDN source

### PR DESCRIPTION
**Pull request by cltnschlosser**
Extra logging for: https://github.com/CocoaPods/CocoaPods/issues/8987

### CODE REVIEW

**Comment by dnkoutso** [comment]
Want to update the commits in this PR it seems it includes a merge commit to master

**Comment by dnkoutso** [comment]
Test 'raises if unexpected HTTP error' failed.

Use `bundle exec bacon <filename>_spec.rb` to run tests locally.

**Comment by cltnschlosser** [comment]
Yeah, I clicked the update button that github suggested :/

Okay, I'll run the tests locally

**Comment by cltnschlosser** [comment]
Should be good now!

File: spec/cdn_source_spec.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@ def cleanup
         should.raise Informative do
           @source.specification('BeaconKit', '1.0.0')
         end.message.
-          should.== "CDN: #{@source.name} Relative path couldn\'t be downloaded: Specs/2/0/9/BeaconKit/1.0.0/BeaconKit.podspec.json Response: 500"
+          should.== "CDN: #{@source.name} Relative path couldn\'t be downloaded: #{@source.url}Specs/2/0/9/BeaconKit/1.0.0/BeaconKit.podspec.json Response: 500"
```
Comment (dnkoutso) [inline spec/cdn_source_spec.rb]: its generally advisable not to use the actual code for assertions as you might end up with a false positive. Recommend putting the actual string here.

Comment (dnkoutso) [reply spec/cdn_source_spec.rb]: Also I see that it does it in the beginning so maybe its fine.

Comment (cltnschlosser) [reply spec/cdn_source_spec.rb]: Makes sense. I was just following the example of `#{@source.name}`.

**Pull request merged**